Casks/h/homerow.rb

```diff
@@ -0,0 +1,27 @@
+cask "homerow" do
+  version "1.2.2"
+  sha256 "463e46c9695857638aa529bf27b3815f2c1fd3babd8887c6e2e5dd7f2584cd02"
+
+  url "https://builds.homerow.app/v#{version}/Homerow.zip"
+  name "Homerow"
+  desc "Keyboard shortcuts for every button on your screen"
+  homepage "https://www.homerow.app/"
+
+  livecheck do
+    url "https://www.homerow.app/appcast.xml"
+    strategy :sparkle, &:short_version
+  end
+
+  depends_on macos: ">= :monterey"
```

Comment:
```suggestion
  auto_updates true
  depends_on macos: ">= :monterey"
```

I would assume based on the presence of sparkle, that this has an in-app update mechanism?

Reply:
Yeah, you're right. It was implied before the previous addition